gh_pages/_source/setup/PC-Setup---ROS.md

```diff
@@ -41,3 +41,49 @@ After this step (or if you already have a working ROS environment), clone the tr
 The following is a quick check to ensure that the appropriate packages have been installed and the the `industrial_training` git repository is current.  Enter the following into the terminal:
 
 `~/industrial_training/.check_training_config.bash`
+
+## VS Code Integration
```

Comment:
Instead of putting this IDE in PC Setup, I would prefer to see a separate page that walks through a few different IDEs (VS Code, Qt Creator) and how to install them.

In particular, saying to install VS Code here would conflict with the actual training workshops we put on, in which we (after this point) have people install Qt Creator.